package.json

```diff
@@ -25,7 +25,16 @@
     "eslint-plugin-node": "^11.0.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
-    "jest": "^24.9.0"
+    "husky": "^4.2.3",
+    "jest": "^24.9.0",
+    "prettier": "^1.19.1",
+    "pretty-quick": "^2.0.1"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged && npm run lint",
```

Comment:
I'm not a fan of extending the time it takes to commit by running additional tasks like linting, testing. I reckon that people should be able to commit how often and whatever they like. It's the CI's job to make sure that the code is valid and properly formatted before it gets into `master`.

Thus, my suggestion is to only run prettier when committing and run linting, tests on CI

However, if you like this approach, you may as well keep it 🙂

Reply:
I will keep it for now and remove it after adding CI to the project.

Reply:
Cool 👍 Do you plan to add CI in the near future 🙂 ?

Reply:
During this week I think.

Reply:
I think I've done that ->https://github.com/mpsujek/codewarssolutions you can see a status badge

Reply:
Yup 😄 It's working 😄 I have suggested a few enhancements in https://github.com/mpsujek/codewarssolutions/issues/35

Reply:
Good job configuring CI, by the way 🙂